src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -68,9 +72,9 @@
     public static final class Launcher
     {
         public static final int kFlywheelMasterId = 7;
-        public static final int kFlywheelFollowerId = -1; // Solid brass
-        public static final int kAngleAdjusterMasterId = 0; // PWM0
-        public static final int kAngleAdjusterFollowerId = 1; // PWM1
+        public static final int kFlywheelFollowerId = 0; // Solid brass
+        public static final int kAngleAdjusterMasterId = 0;
```

Comment:
Add comments again, it's helpful to be able to distinguish these from CAN devices.